### PR TITLE
first file Fahri

### PR DESCRIPTION
**Pull request by fahriozcan**
First week's file is created

### CODE REVIEW

**Comment by AlexandruSendrea** [comment]
Checked it up if it is ok. Seems legit

**Comment by tscselma** [comment]
seems ok

**Comment by Emretester** [comment]
Everything is good

**Comment by xiuhead92** [comment]
all good

**Comment by betulperco** [comment]
well done

File: src/week1/Fahri.java
Comment (lalesi85) [inline src/week1/Fahri.java:1]: It is ok

**Comment by tscselma** [comment]
ok

**Comment by sercankirac** [comment]
ok

**Comment by eyup1312** [comment]
it is gone, what now?

**Comment by tscselma** [comment]
well

**Comment by eyup1312** [comment]
I did not notice that, it is back in town again :)

**Pull request closed without merging**